Code/Editor/Controls/ConsoleSCB.cpp

```diff
@@ -1153,11 +1179,23 @@ void ConsoleVariableEditor::SetVarBlock(CVarBlock* varBlock)
 
 void ConsoleVariableEditor::RegisterViewClass()
 {
-    AzToolsFramework::ViewPaneOptions opts;
-    opts.paneRect = QRect(100, 100, 340, 500);
-    opts.isDeletable = false;
+    if (m_commandInvokedHandler.IsConnected())
+    {
+        m_commandInvokedHandler.Disconnect();
+    }
+    m_commandInvokedHandler.Connect(AZ::Interface<AZ::IConsole>::Get()->GetConsoleCommandInvokedEvent());
```

Comment:
Why not just connect in the ConsoleVariableEditor Constructor?

Reply:
m_commandInvokedHandler handles both the ConsoleVariableEditor case and the DocumentPropertyEditor case. This can get called when ConsoleVariableEditor does not exist.

Reply:
Well that is unfortunate